ledfx/api/scenes_id.py

```diff
@@ -3,23 +3,23 @@
 from aiohttp import web
 
 from ledfx.api import RestEndpoint
-from ledfx.config import find_matching_preset
+from ledfx.config import find_matching_preset, save_config
 from ledfx.utils import generate_id
 
 _LOGGER = logging.getLogger(__name__)
 
 
 class SceneEndpoint(RestEndpoint):
-    """REST end-point for querying a single scene"""
+    """REST end-point for querying and deleting a single scene"""
 
     ENDPOINT_PATH = "/api/scenes/{scene_id}"
 
-    async def get(self, scene_id: str) -> web.Response:
+    async def get(self, scene_id) -> web.Response:
```

Comment:
See if we can try to keep type hints as much as we can - I want to introduce type checking in the future and if we keep adding them as we go it will get a lot nicer.